test/dotnet-test.Tests/GivenThatWeWantToRunTestsInTheConsole.cs

```diff
@@ -95,6 +95,36 @@ public void It_skips_build_when_the_no_build_flag_is_passed()
             result = testCommand.Execute($"{_projectFilePath} -o {_defaultOutputPath} --no-build");
             result.Should().Pass();
         }
+        
+        [Theory]
+        [InlineData("--output", "Missing value for option 'output'")]
+        [InlineData("--configuration", "Missing value for option 'configuration'")]
+        [InlineData("--runtime", "Missing value for option 'runtime'")]
+        [InlineData("--build-base-path", "Missing value for option 'build-base-path'")]
+        public void It_fails_correctly_with_unspecified_arguments_with_long_form(string arguments, string expectedStdErr)
+        {
+            new DotnetTestCommand()
```

Comment:
The form of the test looks the same, the only difference is the argument, can we extract that into a method and remove the duplication?